packages/core/src/index.ts

```diff
@@ -126,6 +144,8 @@ export const Config = Schema.intersect([
       Schema.const(2).description('发送图片、相关信息和链接'),
       Schema.const(3).description('发送全部信息'),
     ]).description('输出方式。').default(1),
+    asset: Schema.boolean().default(false).description('优先使用 assets 服务转存图片。'),
+    base64: Schema.boolean().default(false).description('使用 base64 发送图片。')
```

Comment:
I think it would be better to add document link here that explains how the options work.

Reply:
It would be written in document later.

Reply:
I think our first task is to solve this damm build script and not to get into the minor details.

Reply:
I mean it would be better to add a detailed document and put a link here.